src/ui/public/resize_checker/resize_checker.ts

```diff
@@ -17,41 +17,47 @@
  * under the License.
  */
 
+import { EventEmitter } from 'events';
 import $ from 'jquery';
-import ResizeObserver from 'resize-observer-polyfill';
 import { isEqual } from 'lodash';
-import EventEmitter from 'events';
-
+import ResizeObserver from 'resize-observer-polyfill';
 
-function validateElArg(el) {
+function validateElArg(el: HTMLElement) {
   // the ResizeChecker historically accepted jquery elements,
   // so we wrap in jQuery then extract the element
   const $el = $(el);
 
   if ($el.length !== 1) {
-    throw new TypeError('ResizeChecker must be constructed with a single DOM element.');
+    throw new TypeError(
+      'ResizeChecker must be constructed with a single DOM element.'
+    );
   }
 
   return $el.get(0);
 }
 
-function getSize(el) {
-  return [el.clientWidth, el.clientHeight];
+function getSize(el: HTMLElement | null): [number, number] | null {
```

Comment:
What's the reasoning behind accepting and returning `null`? Why not expect the consumer to avoid calling `getSize(element)` when `element` is null?

Reply:
Actually just to save an additional null check, but I agree, it looks better the other way around, so fixed it. Thanks for pointing out.